dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Abstractions/TopicSubscriptionAttribute.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// TopicSubscriptionAttribute.cs
+
+namespace Microsoft.AutoGen.RuntimeGateway.Grpc.Abstractions;
```

Comment:
This exists in core, looks like it should be removed

Reply:
removed